system/authorized_key.py

```diff
@@ -460,6 +475,7 @@ def main():
            key_options = dict(required=False, type='str'),
            unique      = dict(default=False, type='bool'),
            exclusive   = dict(default=False, type='bool'),
+	   validate_certs = dict(default=True, type='bool'),
```

Comment:
strange why inside gitlab it's shifted by one space, can you try do a pull and check it with VIM?